src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -1352,6 +1352,8 @@ public Metadata updateMetadata(Metadata metadata) {
         String scope;
         if (metadata.getScope().equals(Metadata.GLOBAL_METADATA_SCOPE)) {
             scope = Metadata.GLOBAL_METADATA_SCOPE;
+        } else if (metadata.getScope().startsWith(Metadata.ENTERPRISE_METADATA_SCOPE)) {
```

Comment:
```suggestion
        } else if (metadata.getScope().startsWith(Metadata.ENTERPRISE_METADATA_SCOPE + "_")) {
```

Reply:
This will make it just a little more explicit as to what we're looking for here...

Reply:
Can you bring this logic over to BoxFolder, as well, please?